portfolio/src/main/webapp/index.html

```diff
@@ -11,6 +11,8 @@
         <h1>Abi's Portfolio</h1>
         <button onclick="addRandomQuote()">Click here to see a random quote!</button>
         <div id="greeting-container"></div>
```

Comment:
```suggestion
        <div id="random-quote-container"></div>
```

Nit: rename this to be more relevant to what it displays and make it more distinct from the greet-container (maybe rename that to "greeting-container"?) below